addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1073,6 +1084,37 @@ var SnippetsMenu = Widget.extend({
             }
         });
     },
+    /**
+     * Adds an entry in the left panel box which holds the invisible elements.
+     * The entry will contains an 'Edit' button to activate the snippet.
+     *
+     * @param {jQuery} $invisibleSnippet
+     *        The snippet for which an entry need to be added in the left panel.
+     */
+    _addInvisibleElementsToLeftPanel: function ($invisibleSnippet) {
+        const self = this;
+        $invisibleSnippet.each(function () {
```

Comment:
Use `_.each($invisibleSnippet, el => {` to be able to use `this` instead of `self` (and `el` instead of `this`)